static/js/partners.js

```diff
@@ -1,35 +1,44 @@
-/**
- * Generate a warnin only if warnings are available.
- */
-function warn(message) {
```

Comment:
Are you no longer worried about `console.warn` not existing? Or is this taken care of by moderniser?